powerline/bindings/lemonbar/powerline-lemonbar-multimonitor.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import sys
+import time
+import re
+import subprocess
+
+from threading import Lock, Timer
+from argparse import ArgumentParser
+
+from powerline.lemonbar import LemonbarPowerline
+from powerline.lib.monotonic import monotonic
+from powerline.lib.dict import mergedicts
+
+
+if __name__ == '__main__':
+    parser = ArgumentParser(description='Powerline BAR bindings.')
```

Comment:
ah damn yeah, I started this as just a "personal" file. Actually I need to remove the font settings aswell as they are personal preference/settings here.